src/bin/vip-wp.js

```diff
@@ -199,6 +202,55 @@ const launchCommandAndGetStreams = async ( { guid, inputToken, offset = 0 } ) =>
 	return { stdinStream, stdoutStream, socket };
 };
 
+const bindReconnectEvents = ( { cliCommand, inputToken, subShellRl, commonTrackingParams, isSubShell } ) => {
+	currentJob.socket.io.on( 'reconnect', async () => {
+		debug( '-------- reconnect' );
+		rollbar.info( 'WP-CLI socket.io.on( \'reconnect\' )', { custom: { code: 'wp-cli-on-reconnect', commandGuid: cliCommand.guid } } );
```

Comment:
I thought some rollbar events might be handy here to make sure we're seeing these come through to the client.  What do you think, @luismulinari?

Reply:
It is perfect!